src/components/empty-directory.tsx

```diff
@@ -61,7 +65,7 @@ const EmptyDirectory: React.FC<EmptyFolderProps> = ({ openDialog, theme }) => {
                 <h3>
                     <FormattedMessage id={'emptyDirContent'} />
                 </h3>
-                <Button variant="contained" sx={styles.button} onClick={openDialog} endIcon={<AddIcon />}>
+                <Button variant="contained" sx={styles.button} onClick={onCreateElement} endIcon={<AddIcon />}>
```

Comment:
Well it's the label and the feature of the button into `EmptyDirectory` component.
Ok to use `onButtonClick` prop but we should add a prop to define the button label too to be coherent...
Does this EmptyDirectory component should be this much generic... I don't think so

It's ok for me like this, for example, if a component has a 'clear' button, like MUI DateField, then it offers a `onClear` prop, not a `onButtonClick` which we guess is the clear button (https://mui.com/x/api/date-pickers/date-field/#date-field-prop-onClear)

Reply:
when I see a prop named "onCreateElement", it means it's a callback triggered when an element is created
here, it's a callback to actually create an element if I'm not wrong ?
this is why I find it confusing, it's not because it should match the "onClick" from "<Button>"

Reply:
deal with `onCreateElementButtonClick`